### ISSUE TITLE
Plugin to enable swipe to reply

### ISSUE THREAD

**Issue by Jabdabjoe** [Aliucord/plugin-requests#1402]
### Discord Account

Jabdabjoe

### Plugin Description

A plugin that enables it so a user is able to swipe to reply to a message. Just like how it functions on the current version of un-modded discord.

### Request Agreement

- [X] I have made sure my plugin is possible and abides by the rules!

### ​

- [X] I did indeed check to make sure my plugin request is original!

**Comment by OasisVee** [comment]
not really needed since TapTap exists and personally I think it's so much better than swipe to reply

edit: not sure why it said it twice, only typed it once

**Comment by zf-moth** [comment]
I personally like swiping more, since it's more consistent with other chatting apps

**Comment by Honza572** [comment]
yes please, just make a way to use several gestures, ability to add to them several actions and everyone will be happy 🙏

**Comment by RazerTexz** [comment]
https://github.com/RazerTexz/My-plugins/raw/builds/SwipeToReply.zip
